laterpay/asset_sources/scss/components/_debugger.scss

```diff
@@ -38,29 +40,33 @@
             text-decoration: none;
         }
     }
-}
 
-.lp_debugger-header {
-    background: darken($background--darker, 13%);
-    cursor: pointer;
-    height: round($fs * 2.5);
+
+    // debugger pane header with LaterPay vector logo ------------------------------------------------------------------
+    .lp_debugger-header {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `lp_debugger-header` should be written in lowercase with hyphens